packages/botonic-plugin-contentful/src/cms/contents.ts

```diff
@@ -177,6 +177,12 @@ export class StartUp extends MessageContent {
     ;(clone as any).text = newText
     return clone
   }
+
+  cloneWithButtons(buttons: Button[]): this {
+    const clone = shallowClone(this)
+    ;(clone as any).buttons = buttons
```

Comment:
the semicolon at the beginning?

Reply:
@marcpdw I think this is a "trick" for the compiler to do an inline assignment when using "whatever" `as any`.

Reply:
prettier/eslint inserts it automatically. 
I think it's necessary when we don't use final semicolons and a line starts with a bracket